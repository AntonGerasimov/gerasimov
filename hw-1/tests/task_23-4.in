36 108 11.1
