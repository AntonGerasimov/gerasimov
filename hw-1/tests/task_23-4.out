36
11.100000
l
